homeassistant/components/media_player/channels.py

```diff
@@ -33,9 +33,7 @@
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
-    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-    vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
```

Comment:
You still reference port on line 65. Why remove it ?

Reply:
I was told that they're already in there or something on the last pull. That this was duplicate. So i removed them both. But then i was told i needed `CONF_NAME`, so I put it back. Should I put back port too?

Reply:
I"m not sure why the diff in this pull still shows `CONF_NAME` as being removed :/ I pushed a commit to put it back. Is this diff stale or something?

Reply:
ahhhh never mind, I see it now. It's marked as removed because I did change it when I added it back in, to have better continuation.

Reply:
My comment in the previous PR was on the default value in the config dict lookup inside `setup_platform`. You should remove that, not this any of the lines in the config validation schema.